pelican/readers.py

```diff
@@ -264,7 +278,7 @@ def _parse_metadata(self, meta):
                 self._md.reset()
                 formatted = self._md.convert(formatted_values)
                 output[name] = self.process_metadata(name, formatted)
-            elif name in METADATA_PROCESSORS:
+            elif not DUPLICATES_DEFINITIONS_ALLOWED.get(name, True):
```

Comment:
If all of the values in `DUPLICATES_DEFINITIONS_ALLOWED` are `False`, wouldn't it be cleaner to just refactor that dictionary into a set, and check for set membership here?

Reply:
I don't think so, explicit is better.

Having an attribute not in the dict would clearly say "we still did not think of it".

Having an attribute set to True would clearly mean "We though of it, and we explicitly allow for duplicates, in the future, do not spend time thinking if it's OK or not".

Yes I imagine this dict growing over time to explicitly tell if something is allowed to appear multiple times or not.